packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/google-drive.tsx

```diff
@@ -36,7 +43,10 @@ const GoogleDrive = ({ is_authorised, is_open_button_loading, onDriveConnect, on
                         <Button text={localize('Disconnect')} onClick={onDriveConnect} has_effect secondary large />
                         <Button
                             text={localize('Open')}
-                            onClick={onDriveOpen}
+                            onClick={() => {
+                                onDriveOpen();
+                                setOpenSettings('import', true);
```

Comment:
please make the 2nd parameter default to `true` so we only need to pass 2nd parameter when it is false.

Reply:
@prince-deriv if we set this to default `true` then it will be visible on page load. So we cannot set it to default true